### DISCUSSION TITLE
Command to upgrade node in nvm windows..

### DISCUSSION THREAD

**Discussion by JenuelDev** [coreybutler/nvm-windows#684] [Support] [answered]
Is there a way to upgrade node?

**Comment by coreybutler** [comment, marked as answer]
No, you just install whatever version you want.

There is no official process for upgrading Node without a version manager. Some people have requested the ability to upgrade global modules (which some version managers do). The problem with automatic upgrades is some global npm modules break when upgrading/downgrading.

This is something I intend to address in rt (see [rt details](https://github.com/coreybutler/nvm-windows/discussions/565#discussioncomment-58112)).
